libphal/phal_exception.H

```diff
@@ -25,6 +25,7 @@ enum ERR_TYPE {
 	SBE_CMD_TIMEOUT,
 	SBE_FFDC_NO_DATA,
 	SBE_BOOT_SEEPROMS_BAD,
+	SBE_MEASUREMENT_SEEPROMS_BAD,
```

Comment:
**Commit message:**

> Adding support for recovery action REIPL_BKP_MSEEPROM/
REIPL_UPD_MSEEPROM (reipl using backup measurement seeprom).

Please update what needs to be done for the REIPL_BKP_MSEEPROM and REIPL_UPD_MSEEPROM recovery action in the commit message.